homeassistant/components/aseko_pool_live/sensor.py

```diff
@@ -18,6 +22,67 @@
 from .entity import AsekoEntity
 
 
+@dataclass(frozen=True, kw_only=True)
+class AsekoSensorEntityDescription(SensorEntityDescription):
+    """Describes an Aseko binary sensor entity."""
+
+
+CONCENTRATION_KILOGRAMS_PER_CUBIC_METER = "kg/m³"
+CONCENTRATION_MILLIGRAMS_PER_LITER = "mg/l"
+GRAMS_PER_HOUR = "g/hour"
+
+UNIT_SENSORS = {
```

Comment:
Are these all the entity descriptions? Do we expect more?

Reply:
Unfortunately there are more, but I do not have access to all types of the Aseko units, so I left the fallback not to break other types of entities for other types of units (non-Salt variants):
```
        else:
            self._attr_name = self._variable.name
            self._attr_native_unit_of_measurement = self._variable.unit
```

Reply:
Do you know any other users who use this? Can we predict some? Maybe we can check some GitHub issues for people who can test. Making this list exhausted is a big plus